pages/docs/core-concepts/_meta.json

```diff
@@ -1,5 +1,5 @@
 {
-  "effect": "Effect",
+  "create-and-run-effects": "Create and Run Effects",
```

Comment:
Maybe let's just name this to `creating-effects` and we can include examples of running the Effects themselves within the documentation.